src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +41,51 @@
      * Андрей - 600
      * София - 900</p>
      */
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
make it constant

Reply:
Can I write it in lowercase? Maven doesn't let me make it constant in all capital letters :(